frontend/packages/knative-plugin/src/components/add/event-sources/ApiServerSection.tsx

```diff
@@ -62,7 +62,7 @@ const ApiServerSection: React.FC<ApiServerSectionProps> = ({ namespace }) => {
   return (
     <FormSection title="ApiServerSource">
       <FormGroup fieldId={fieldId} label="Resource" isRequired>
-        <NameValueEditor
+        <NameValueEditorComponent
```

Comment:
Why not use `AsynCOmponent` directly here instead of creating this new wrapper component?

Reply:
Yeah I added a separate one as it's used in two of the forms under sources so this way thought of using same wrapper both places.